### ISSUE TITLE
Terminal Crashes

### ISSUE THREAD

**Issue by shawonis08** [microsoft/vscode#97678]
When I use multiple terminals after a certain period of time it crashes the vs code or stop working with exit code: 1
![vs_code_terminal_issue_report](https://user-images.githubusercontent.com/15838889/81779071-ccbcd280-9515-11ea-8d08-4bc84b017e7e.jpg)

**Comment by Tyriar** [comment]
Can you run `ver` in cmd.exe to get your windows version and build number?

**Comment by shawonis08** [comment]
Microsoft Windows [Version 10.0.18362.778]

**Comment by Tyriar** [comment]
And when you say

> after a certain period of time it crashes

Do you mean it just happens randomly all of a sudden, or is it when you're entering something into the terminal? I haven't seen anything like this happen personally.

**Comment by shawonis08** [comment]
yes, randomly & especially when I work with python3.6 & node application together in multiple terminals.

**Comment by handledexception** [comment]
I am experiencing an issue like this on `Microsoft Windows [Version 10.0.18363.900]`.

VSCode About window:
```
Version: 1.46.0 (user setup)
Commit: a5d1cc28bb5da32ec67e86cc50f84c67cc690321
Date: 2020-06-10T09:03:20.462Z
Electron: 7.3.1
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363.
```

I wouldn't say these are reliable repro steps but coincidentally it seems to occur more often after I've done these things:

1. Open the integrated terminal w/ cmd.exe
2. Start typing a command of some sort into the terminal but don't execute it. For example I start typing my git commit -m message but don't submit yet.
3. Grab the horizontal separator over the top of the integrated terminal and start dragging it up and down to resize the console.

Eventually the integrated terminal will terminal with exit code 1.

**Comment by Tyriar** [comment]
I cannot reproduce this on my machine unfortunately.
